components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -93,28 +78,62 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem {
   _selectedItem = selectedItem;
 }
 
-#pragma mark - UIView
+- (void)setUpSubviews {
```

Comment:
Please rename this to `commonMDCTabBarViewInit`.

Reply:
Fix applied.